src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/LineLengthCheck.java

```diff
@@ -105,6 +111,41 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * To configure the check to validate the use of check on import statements.
+ * </p>
+ * <pre>
+ * &lt;module name="LineLength"&gt;
+ *   &lt;property name="validateImport" value="true"/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * import com.puppycrawl.tools.checkstyle.grammar.InputFullOfSinglelineComments; // Violation
+ * import static com.puppycrawl.tools.checkstyle.grammar
+ *           .comments.InputFullOfSinglelineComments.main;
+ *                                                 ^-- // ok
```

Comment:
Yes it can be, I had the wrong impression that default max length is 60. The check also considers comments as a part of the line so to keep it under limit I typed it on the next line. But the max default is 80 so it shall be fine now